heppy/Daemon.py

```
@@ -48,10 +47,10 @@ def quit(self):
         quit()
 
     def hello(self):
-        print "HELLO"
+        print("HELLO")
         self.force_hello = True
 
-    def start(self, args = {}):
+    def start(self, args={}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)